array.js

```diff
@@ -0,0 +1,11 @@
+const oldEmployees = ["Mike", "Nabil", "Cubby", "Mendes"];
+const newEmployees = ["Marcus", "Rasmus", "Matt"];
+const allEmployees = [];
+
+oldEmployees.forEach((employee) => {
+  allEmployees.push(employee);
+});
+
```

Comment:
Hey man, nice work but there's a simpler way of solving this. Try using the ES6 spread operator instead! It will reduce your code down to only three lines!

const allEmployees = [...oldEmployees, ...newEmployees]